[Bug][Manager] Elasticsearch port parameter is missing

### What happened

Elasticsearch `port` parameter is missing.

### What you expected to happen

First, elasticsearch `port` parameter is missing.

Table options are:
'connector'='elasticsearch-6-inlong'
'document-type'='null'
'hosts'='http://x.x.x.x:null'
'index'='test_index_10'
'inlong.metric.labels'='groupId=group_10&streamId=stream_01&nodeId=sink_01'
'metrics.audit.proxy.hosts'='x.x.x.x:10081'
'password'='xxx'
'username'='xxx'
 ...
Caused by: org.apache.flink.table.api.ValidationException: Could not parse host 'http://x.x.x.x:null' in option 'hosts'. It should follow the format 'http://host_name:port'.
	at org.apache.inlong.sort.elasticsearch6.table.Elasticsearch6Configuration.validateAndParseHostsString(Elasticsearch6Configuration.java:75) ~[?:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_151]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1380) ~[?:1.8.0_151]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_151]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_151]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_151]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_151]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499) ~[?:1.8.0_151]
	at org.apache.inlong.sort.elasticsearch6.table.Elasticsearch6Configuration.getHosts(Elasticsearch6Configuration.java:44) ~[?:?]

Then, in mysql table `stream_sink`, the  `port` parameter is also null.

This parameter `port` is configured on the dashboard, like this:
![image](https://user-images.githubusercontent.com/105137931/206888808-13c0a3f6-e8ed-4097-ae05-a34ca6d597b8.png)


### How to reproduce

This problem will reproduce when configuring tasks written to es on the dashboard. 

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/es/ElasticsearchSinkDTO.java
+++ b/inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/es/ElasticsearchSinkDTO.java
@@ -95,6 +95,7 @@ public class ElasticsearchSinkDTO {
         }
         return ElasticsearchSinkDTO.builder()
                 .host(request.getHost())
+                .port(request.getPort())
                 .username(request.getUsername())
                 .password(passwd)
                 .indexName(request.getIndexName())

